modules/nixos-wiki/backup.nix

```diff
@@ -64,17 +83,95 @@ in
     };
   };
 
-  systemd.services.wiki-restore = {
+  systemd.services.old-wiki-restore = {
     startAt = "daily";
     path = [ pkgs.postgresql mediawiki-maintenance ];
 
     serviceConfig = {
-      ExecStart = "${wiki-restore}/bin/wiki-restore";
+      ExecStart = "${old-wiki-restore}/bin/old-wiki-restore";
+      Type = "oneshot";
+    };
+  };
+
+  systemd.services.wiki-backup = {
```

Comment:
Instead of service, should this not run just in the preHook of the backup service?
Than we the state of the backup and uploads are closer together.